dev_guide/builds.adoc

```diff
@@ -1575,6 +1588,12 @@ are ignored. Set the `*allowEnv*` field to `*true*` on the webhook definition to
 this behavior.
 ====
 
+To pass this payload using `*curl*`, define it in a file named `*payload_file.yaml*` and run:
```

Comment:
only backticks around curl (no asterisks), as it is command
Change formatting of payload_file.yaml to:

     *_payload_file.yaml_*